### PR TITLE
post(monitoring):  enhancing production monitoring production with newrelic

### PR DESCRIPTION
**Pull request by MarineJu**
This PR adds a new post as well as its two new authors Marwa Ben Ali and Marine Julieron with their avatar

### CODE REVIEW

File: _posts/2024-09-09-enhancing-production-monitoring-with-newrelic.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
+---
+layout: post
+title: Enhancing Production Monitoring with New Relic
+description: How did we improve the monitoring practices on the mobile service ? Let's find out.
```
Comment (qtomasicchio) [inline _posts/2024-09-09-enhancing-production-monitoring-with-newrelic.md:4]: I don't think we have a space before a `?` in english
```suggestion
description: How did we improve the monitoring practices on the mobile service? Let's find out.
```

File: _posts/2024-09-09-enhancing-production-monitoring-with-newrelic.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+language: en
+thumbnail: "/images/posts/2024-09-09-enhancing-production-monitoring-with-newrelic/main.png"
+---
+
+## Enhancing Production Monitoring with New Relic 
+ 
+At *Bedrock*, we develop a streaming solution tailored for European media companies. Our application is a customizable white-label product used by millions across Europe. Bedrock’s clients have high expectations for stability, early incident detection, quick resolution, and effective communication during those times. Naturally, our goal is to deliver a stable product, and we dedicate significant effort toward this objective. 
+
+However, about two and a half years ago, our mobile team faced serious challenges in monitoring production. The tagging systems were pushing events into a tool that developers were not kin to use. The graphs were not intuitive and difficult to generate, making them hard to interpret. Each team across the company was managing its own monitoring practices. Monitoring was largely manual.  
+Too many problems were brought to the attention of the team by a complain of our client to the support team. We knew we had to make a change.
```
Comment (qtomasicchio) [inline _posts/2024-09-09-enhancing-production-monitoring-with-newrelic.md:17]:
```suggestion
Too many problems were brought to the attention of the team by a complaint of our client to the support team. We knew we had to make a change. 
```

File: _posts/2024-09-09-enhancing-production-monitoring-with-newrelic.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+
+ ```
+ SELECT count(*) 
+ FROM Actions 
+ WHERE actionName = 'LoginAction' 
+ FACET status, authenticationMethod 
+ TIMESERIES auto 
+ ```
+
+This query counts the number of occurrences of the 'LoginAction' event, faceting by the status and authenticationMethod fields. The TIMESERIES auto clause visualizes this count over time, showing trends or spikes in login activity.
```
[the remark this answers is not in the document]
Comment (qtomasicchio) [reply _posts/2024-09-09-enhancing-production-monitoring-with-newrelic.md:58]: Is it `faceting` or `faceted` in the context of the phrase? I'm not sure 🤔

**Review by qtomasicchio — APPROVED** [review]
Nice article! 💯

File: _posts/2024-09-09-enhancing-production-monitoring-with-newrelic.md [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+## Setting Our Goals and Expectations 
+
+We began by defining our goals: 
+
+* Confidence in Releases: We needed certainty that what we shipped was functioning as expected. 
+* Standardized Practices: It was crucial to align monitoring practices and tagging plans across teams. 
+* Early Problem Detection: Our goal was to identify issues as early as possible. 
+* Quick Data for Support Teams: We aimed to swiftly provide support with relevant metrics, such as the number of users impacted and the duration of incidents. 
+* Developer Adoption: The selected tool had to be user-friendly, offering easy graph creation, visually appealing layouts, straightforward querying, and an ergonomic interface. 
+* Automated Alerts: We needed a system that could automatically alert us to potential issues.
```
[the remark this answers is not in the document]
Comment (MarineJu) [reply _posts/2024-09-09-enhancing-production-monitoring-with-newrelic.md:30]: i think it is of

**Pull request merged**